eosdis-nasa/earthdata-pub-forms
# Edpub 1092 forms string units together

# Description
Adds jest unit testing to forms

# Spec
See Ticket: https://bugs.earthdata.nasa.gov/browse/EDPUB-1092

# Validation
1. Pull related branches locally.
2. Install dependencies
3. Run tests using `npm run test:unit`
4. Verify that test have succeeded
